docs/admin/install/troubleshoot.rst

```diff
@@ -1,6 +1,54 @@
 Troubleshooting Installation Errors
 ===================================
 
+Error importing submission key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If importing the submission key  using ``sdw-admin --configure`` fails, you can also copy the submission key manually.
+
+- Open a ``dom0`` terminal by opening the **Q Menu**, selecting the gear icon on the left-hand side, then selecting **Other > Xfce Terminal**. Once the Terminal window opens, run the following command to list the SVS submission key details, including its fingerprint:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault \
+      "gpg --homedir /run/media/user/TailsData/gnupg -K --fingerprint"
+
+- Next, run the comand:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault \
+      "gpg --homedir /run/media/user/TailsData/gnupg --export-secret-keys --armor <SVSFingerprint>" \
+      > /tmp/sd-journalist.sec
+
+  where ``<SVSFingerprint>`` is the submission key fingerprint, typed as a single unit without whitespace. This will copy the submission key in ASCII format to a temporary file in dom0, ``/tmp/sd-journalist.sec``.
+
+- Verify the that the file starts with ``-----BEGIN PGP PRIVATE KEY BLOCK-----`` using the command:
+
+  .. code-block:: sh
+
+    head -n 1 /tmp/sd-journalist.sec
+
+- Unmount the SVS USB 
```

Comment:
This leaves users with the file sitting in `/tmp`, but not with a working import. This would be a good place to move the manual instructions that are not necessary with `--configure`.